fp/_baseConvert.js

```diff
@@ -279,6 +279,9 @@ function baseConvert(util, name, func, options) {
       setPlaceholder = true;
       func.placeholder = result.placeholder = placeholder;
     }
+    result.convert = function(options) {
+      return baseConvert(util, name, func, options);
```

Comment:
But now I see that it remembers the last customization rules. Isn't that a bad thing?

Let's assume all the default rules are true for the function `fun`

```js
var fun1 = require("lodash/fp/fun").convert({cap: false})
var fun2= require("lodash/fp/fun").convert({rearg: false})
```
I would expect that `fun2` has `cap:true` but it seems that it would be `false`?